src/main/java/billgates/view/gui/ActionLabel.java

```diff
@@ -0,0 +1,18 @@
+package billgates.view.gui;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class ActionLabel extends JLabel {
+
+    public static final int DEFAULT_FONT_SIZE = 13;
+    public static final Font DEFAULT_FONT = new Font("Modern No. 20", Font.BOLD, DEFAULT_FONT_SIZE);
+
+    public static final Color DEFAULT_Text_COLOR = Color.GRAY;
```

Comment:
I don't know why this is "DEFAULT_**Text**_COLOR" instead of it being ALL CAPS. XD